lib/hpcbase.pm

```diff
@@ -347,44 +347,53 @@ from the rest and can be exported via a network file system.
 
 After C<prepare_spack_env> run, C<spack> should be ready to build entire tool stack,
 downloading and installing all bits required for whatever package or compiler.
+
+This sub is designed to install one of the mpi implementations. Although there are
+thousands packages to be used.
+
+LD_LIBRARY_PATH is removed and spack is not exported. In case LD_LIBRARY_PATH is required
+it has to be add it in the F<.spack/modules.yaml>.
+
+=begin text
+  See bsc#1208751 for details
+=end text
+
+To do that run
+
+=begin text
+  spack config add modules:prefix_inspections:lib64:[LD_LIBRARY_PATH]
+  spack config add modules:prefix_inspections:lib:[LD_LIBRARY_PATH]
+=end text
+
 =cut
 
 sub prepare_spack_env {
     my ($self, $mpi) = @_;
     $mpi //= 'mpich';
+    zypper_call "in libopenssl-devel", timeout => 480;
```

Comment:
You can install all in one `zypper_call` together on line 384.

Reply:
moreover, why it is needed? `ibopenssl-devel` is already pulled by spack as dependency

Reply:
this is messed up from squash. I will remove it

Reply:
fixed and ready for review